forte/processors/data_augment/algorithms/eda_ops.py

```diff
@@ -278,63 +281,68 @@ def default_configs(cls):
         return {
             "augment_entry": "ft.onto.base_ontology.Token",
             "other_entry_policy": {
-                # to use Texar hyperparams 'kwargs' must
-                # accompany with 'type'
                 "ft.onto.base_ontology.Document": "auto_align",
                 "ft.onto.base_ontology.Sentence": "auto_align",
             },
             "alpha": 0.1,
-            "augment_pack_names": {
-                "input_src": "augmented_input_src",
-            },
         }
 
 
-class RandomInsertionDataAugmentProcessor(ReplacementDataAugmentProcessor):
+class RandomInsertionDataAugmentOp(BaseDataAugmentationOp):
     r"""
     Data augmentation processor for the Random Insertion operation.
     Find a random synonym of a random word in the sentence that is
     not a stop word. Insert that synonym into a random position in
     the sentence. Do this n times, where n = alpha * input length.
     """
 
-    def initialize(self, resources: Resources, configs: Config):
-        super().initialize(resources, configs)
-        # pylint: disable=attribute-defined-outside-init
-        self.stopwords = set(configs["stopwords"])
+    def augment(self, data_pack: DataPack) -> bool:
 
-    def _augment(self, input_pack: MultiPack, aug_pack_names: List[str]):
         replacement_op = create_class_with_kwargs(
-            self.configs["data_aug_op"],
-            class_args={"configs": self.configs["data_aug_op_config"]},
+            self.configs["insertion_op"],
+            class_args={"configs": self.configs["insertion_op_config"]},
         )
         augment_entry = get_class(self.configs["augment_entry"])
 
-        for pack_name in aug_pack_names:
-            data_pack: DataPack = input_pack.get_pack(pack_name)
-            annotations: List[Annotation] = []
-            pos = [0]
+        annotations: List[Annotation] = []
+        pos = [0]
+        try:
             annos: Iterable[Annotation] = data_pack.get(augment_entry)
             for anno in annos:
-                if anno.text not in self.stopwords:
+                if anno.text not in set(self.configs["stopwords"]):
```

Comment:
If we want to convert to `set`, we should do it during `initialize` instead of during every op.

Reply:
Can I do it in the `__init__` method instead? We don't use `initialize` in the `BaseDataAugmentationOp`

Reply:
yeah, just same spirit